watch.sh

```diff
@@ -0,0 +1 @@
+find . -type d | python -c "import sys; import json; dirs = [d[2:-1] + '/' for d in sys.stdin]; del dirs[0]; sys.stdout.write(json.dumps({'folders': dirs}))" > scripts/folders.json 
```

Comment:
ya im happy with it! runs super quick too. super lucky that the find command basically outputs exactly what we want